### PR TITLE
Batch update onMobDeath->onMobDespawn PH's & NM's

### PR DESCRIPTION
**Pull request by ymkc**
This contains all the cheap & easy Leaping Lizzy like NM's and PH's

### CODE REVIEW

**Comment by TeoTwawki** [comment]
1 small caveat: the `setDropRate()` commands won't work there. Everything else is good that I can see.

Actually, I am near certain they should never have been in onMobDeath anyway and cannot work in either function, so I guess this doesn't break anything! I'm not sure they could be moved into `onMobSpawn()` but I've been meaning to move them to sql anyway (a change to the table to handle drops in "slots" is required, but I screwed up my branch I was doing that in :frowning: ).

**Comment by TeoTwawki** [comment]
tl:dr I say merge as is anyway, coz those lines days are numbered

File: scripts/zones/Valkurm_Dunes/mobs/Valkurm_Emperor.lua [hunk 1]
```diff
@@ -4,10 +4,10 @@
 -----------------------------------
 
 -----------------------------------
--- onMobDeath
+-- onMobDespawn
 -----------------------------------
 
-function onMobDeath(mob, player, isKiller)
+function onMobDespawn(mob)
```
Comment (takhlaq) [inline scripts/zones/Valkurm_Dunes/mobs/Valkurm_Emperor.lua]: there's two onMobDespawns in this file doing the exact same thing

Comment (takhlaq) [inline scripts/zones/Valkurm_Dunes/mobs/Valkurm_Emperor.lua]: other than that its gud

Comment (ymkc) [inline scripts/zones/Valkurm_Dunes/mobs/Valkurm_Emperor.lua]: didn't catch that, fixed

**Pull request merged**